### PR TITLE
Anouar components

### PR DESCRIPTION
**Pull request by Nouri-Anouar**
Components:

- OpenResponse 
- FormField
- Slider

### CODE REVIEW

**Review by Enjoy2Live — CHANGES_REQUESTED** [review]
Amazing work @Nouri-Anouar , please go over the requested changes and let me know what you think in the comments here,
I don't see OpenResponse stories in this PR and I don't know if we still going to need FormField?

P.S. congrats on Morocco winning against Belgium!

File: packages/UI/src/components/organisms/OpenResponse/StyledOpenResponse.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import styled, { css } from 'styled-components';
+import { typographyStyles } from '../../atoms/Typography/Styled.Typography';
+import type { OpenResponseProps } from '.';
+export const Label = styled.label`
+  ${typographyStyles['label']};
+  margin-left: 1rem;
```
Comment (Enjoy2Live) [inline packages/UI/src/components/organisms/OpenResponse/StyledOpenResponse.ts:6]: @Nouri-Anouar why are we doing margin left of 1 rem?

File: packages/UI/src/components/atoms/Slider/Slider.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  LabelsContainer,
+  SliderInput,
+  SliderInputContainer,
+} from './StyledSlider';
+import type { SliderProps } from '.';
+
+export default function Slider({
+  min = 0,
+  max = 100,
+  prefix,
```
[the remark this answers is not in the document]
Comment (Nouri-Anouar) [reply packages/UI/src/components/atoms/Slider/Slider.tsx]: I agree with you. changed it.

File: packages/UI/src/components/atoms/Slider/StyledSlider.ts [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  color: ${({ theme }) => theme.colors.GREYSCALE_BLACK};
+  max-width: 360px;
+  user-select: none;
+`;
+
+export const SliderInputContainer = styled.div`
+  box-sizing: border-box;
+  width: 100%;
+  margin: 0;
+  @media (max-width: 760px) {
```
[the remark this answers is not in the document]
Comment (Nouri-Anouar) [reply packages/UI/src/components/atoms/Slider/StyledSlider.ts]: This was not necessary and removed it

File: packages/UI/src/components/atoms/Slider/Slider.tsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  };
+
+  return (
+    <Container>
+      <BubbleContainer>
+        <Bubble newVal={getBubblePosition()}>
+          <p>
+            <span>{value}</span>
+            <span>{prefix}</span>
+          </p>
```
[the remark this answers is not in the document]
Comment (Nouri-Anouar) [reply packages/UI/src/components/atoms/Slider/Slider.tsx]: can you tell me what is wrong @Enjoy2Live 
I am trying to use this import 
`import { atoms } from '@devlaunchers/components/src/components';`
but I get this error
`Cannot find module '@devlaunchers/components/src/components'`

[the remark this answers is not in the document]
Comment (Enjoy2Live) [reply packages/UI/src/components/atoms/Slider/Slider.tsx]: @Nouri-Anouar where you are you trying to import that?

[the remark this answers is not in the document]
Comment (Nouri-Anouar) [reply packages/UI/src/components/atoms/Slider/Slider.tsx]: components/atoms/Slider/Slider.tsx

[the remark this answers is not in the document]
Comment (Enjoy2Live) [reply packages/UI/src/components/atoms/Slider/Slider.tsx]: you can't import it that way cuz `@devlaunchers/components` is the library itself, this format of importing only works on external apps that have `@devlaunchers/components` as dependency in their `package.json`.

you can import the components you want relatively, get used to the dots!

[the remark this answers is not in the document]
Comment (Nouri-Anouar) [reply packages/UI/src/components/atoms/Slider/Slider.tsx]: I see, that makes sense.
Thank you!

File: packages/UI/src/components/organisms/OpenResponse/StyledOpenResponse.ts (hunk 1 above)
Comment (Nouri-Anouar) [reply packages/UI/src/components/organisms/OpenResponse/StyledOpenResponse.ts]: I solved that issue

File: packages/UI/src/components/molecules/BackButton/StyledBackButton.ts [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  display: inline-flex;
+  align-items: center;
+  gap: 8px;
+  border-radius: ${({ type }) => (type === 'cancel' ? '4px' : '8px')};
+  padding: ${({ type }) => (type === 'cancel' ? '8px' : '8px 16px')};
+  background: ${({ theme }) => theme.colors.GREYSCALE_WHITE};
+  ${({ type }) =>
+    type === 'cancel'
+      ? shadowStyles.shadow500
+      : 'box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25)'};
```
[the remark this answers is not in the document]
Comment (Nouri-Anouar) [reply packages/UI/src/components/molecules/BackButton/StyledBackButton.ts:17]: @Enjoy2Live Good point!
I just copied it directly from BackButton styles in Figma and indeed it does not exist in shadows.
Thank you for asking her.

File: packages/UI/src/components/atoms/Slider/StyledSlider.ts [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+`;
+
+export const SliderInput = styled.input.attrs({ type: 'range' })`
+  -webkit-appearance: none;
+  overflow: hidden;
+  width: 100%;
+  height: 14px;
+  background: ${({ theme }) => theme.colors.GREYSCALE_OFF_WHITE};
+  outline: 2px solid black;
+  border-radius: 50px;
```
[the remark this answers is not in the document]
Comment (Nouri-Anouar) [reply packages/UI/src/components/atoms/Slider/StyledSlider.ts:23]: Yes, 3rem is good for that, are you going to change that?

**Pull request merged**